products/flux2.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Flux
+category: server-app
+tags: cncf
+iconSlug: flux2
+permalink: /flux
+alternate_urls:
+-   /flux2
+-  /fluxcd
+versionCommand: flux version
+releasePolicyLink: https://fluxcd.io/flux/releases/#supported-releases
+changelogTemplate: https://github.com/fluxcd/flux2/releases/tag/v__LATEST__
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+-   git: https://github.com/fluxcd/flux2.git
+
+# eol(X) = releaseDate(X+3)
+releases:
+-   releaseCycle: "2.0"
+    releaseDate: 2023-07-05
+    eol: false
+    latest: "2.0.1"
+    latestReleaseDate: 2023-07-11
+
+---
+
+> [Flux](https://fluxcd.io) is an open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
```

Comment:
```suggestion
> [Flux](https://fluxcd.io) Flux is a tool for keeping Kubernetes clusters in sync with sources of configuration (like Git repositories), and automating updates to configuration when there is new code to deploy.
> Flux is built from the ground up to use Kubernetes' API extension system, and to integrate with Prometheus and other core components of the Kubernetes ecosystem. It supports multi-tenancy and syncing an arbitrary number of Git repositories.
```

What does `Powered by GitOps Toolkit.` means ? I suggest to use the description on https://fluxcd.io/flux/.